build_projects/shared-build-targets-utils/Utils/Monikers.cs

```diff
@@ -93,6 +71,11 @@ public static string GetSdkDebianPackageName(BuildTargetContext c)
             return $"{packagePrefix}-dev-{nugetVersion}";
         }
 
+        public static string GetDebianHostFxrPackageName(BuildTargetContext c)
```

Comment:
Let's remove it if it isn't called.

core-setup should have it's own "HostMonikers" class.  I really don't like putting non-shared things in this 'shared-utils' and then trying to keep them in sync.  Anyone coming in here will see this dead code and will remove it.